example/package.json

```diff
@@ -15,7 +15,7 @@
     "react-dom": "18.2.0",
     "react-native": "0.71.7",
     "react-native-web": "~0.18.10",
-    "react-native-webview": "^12.0.2"
+    "react-native-webview": "11.26.0"
```

Comment:
Why downgrade?

Reply:
because the error version mismatch and RTE does not work